attest/index.js

```diff
@@ -0,0 +1,80 @@
+import core from '@actions/core';
+import exec from '@actions/exec';
+import artifact from '@actions/artifact';
+import path from 'path';
+
+function getActionConfig() {
+    return {
+        cimon: {
+            logLevel: core.getInput('logLevel'),
+            clientId: core.getInput('clientId'),
+            secret: core.getInput('secret'),
+            url: core.getInput('url'),
+            releasePath: core.getInput('releasePath'),
+        },
+        attest: {
+            subjects: core.getInput('subjects'),
+            imageRef: core.getInput('imageRef'),
+            signKey: core.getInput('signKey'),
+            provenanceOutput: core.getInput('provenanceOutput'),
+            signedProvenanceOutput: core.getInput('signedProvenanceOutput'),
+            githubContext: core.getInput('githubContext'),
+        },
+        report: {
+            reportJobSummary: core.getInput('reportJobSummary'),
+            reportArtifact: core.getInput('reportArtifact'),
```

Comment:
I think you can use `core.getBoolInput('reportArtifact')` and remove the code converting it to bool (or checking with `true`)